src/main/java/teammates/ui/request/FeedbackSessionCreateRequest.java

```diff
@@ -1,19 +1,41 @@
 package teammates.ui.request;
 
+import javax.annotation.Nullable;
+
 /**
  * The request body format for creation of feedback session.
  */
 public class FeedbackSessionCreateRequest extends FeedbackSessionBasicRequest {
     private String feedbackSessionName;
+    @Nullable
+    private String toCopyCourseId;
+    @Nullable
+    private String oldSessionName;
 
     public String getFeedbackSessionName() {
         return feedbackSessionName;
     }
 
+    public String getToCopyCourseId() {
+        return toCopyCourseId;
+    }
+
+    public String getOldSessionName() {
+        return oldSessionName;
+    }
+
     public void setFeedbackSessionName(String feedbackSessionName) {
         this.feedbackSessionName = feedbackSessionName;
     }
 
+    public void setToCopyCourseId(String toCopyCourseId) {
+        this.toCopyCourseId = toCopyCourseId;
+    }
+
+    public void setOldSessionName(String oldSessionName) {
+        this.oldSessionName = oldSessionName;
+    }
+
     @Override
     public void validate() {
```

Comment:
We need to update the validation - I think if `toCopyCourseId` is not null then `oldSessionName` should not be null as well?

Can we also rename `oldSessionName` to `toCopySessionName`?

Reply:
I have updated the validation and renamed to `toCopySessionName`